src/analysis/complex/basic.lean

```diff
@@ -60,14 +60,14 @@ lemma dist_eq (z w : ℂ) : dist z w = abs (z - w) := rfl
 
 @[simp] lemma norm_real (r : ℝ) : ∥(r : ℂ)∥ = ∥r∥ := abs_of_real _
 
-@[simp] lemma norm_rat (r : ℚ) : ∥(r : ℂ)∥ = _root_.abs (r : ℝ) :=
-suffices ∥((r : ℝ) : ℂ)∥ = _root_.abs r, by simpa,
+@[simp] lemma norm_rat (r : ℚ) : ∥(r : ℂ)∥ = has_abs.abs (r : ℝ) :=
+suffices ∥((r : ℝ) : ℂ)∥ = has_abs.abs r, by simpa,
```

Comment:
Why aren't you using the notation you introduced here?

Reply:
I was probably saving it for the other PR https://github.com/leanprover-community/mathlib/pull/8891 . I've updated this PR to use the notation now.